specifyweb/frontend/js_src/lib/components/WbImport/helpers.ts

```diff
@@ -160,9 +161,9 @@ const MAX_NAME_LENGTH = 64;
 export async function uniquifyDataSetName(
   name: string,
   currentDataSetId?: number,
-  datasetsUrl = '/api/workbench/dataset/'
+  datasetsUrl: keyof typeof datasetVariants = 'workbench'
```

Comment:
you have WB_VARIANT (WbVariant) type in a different file. should you export it and use it here?